maps/biter_battles_v2/main.lua

```diff
@@ -186,6 +186,7 @@ end
 
 local function on_marked_for_deconstruction(event)
 	if not event.entity.valid then return end
+	if not event.player_index == nil then return end
```

Comment:
```suggestion
	if not event.player_index then return end
```
sorry, your fix doesn't work

Reply:
Thanks a lot, you are right, oh god... That's a good example of why review are important sometimes :D